recipes/homog2d/all/conanfile.py

```diff
@@ -28,9 +24,11 @@ def _min_cppstd(self):
     @property
     def _compilers_minimum_version(self):
         return {
-            "gcc": "5",
+            "gcc": "6",
```

Comment:
Why was this bumped if `min_cppstd` is still 14? (I also have questions about `apple-clang`) I'd appreciate an explanation so it's easier for me to understand next time around :)

Reply:
@RubenRBS 
Since homog2d/2.10 has compilation error on gcc 5, I decided to upgrade minimum version of gcc to 6.

https://c3i.jfrog.io/c3i/misc/logs/pr/16158/1-linux-gcc/homog2d/2.10//5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9-test.txt

Reply:
Thanks for the clarification!